pkg/pxc/app/statefulset/node.go

```diff
@@ -119,6 +119,10 @@ func (c *Node) AppContainer(spec *api.PodSpec, secrets string, cr *api.PerconaXt
 				Name:      "ssl-internal",
 				MountPath: "/etc/mysql/ssl-internal",
 			},
+			{
+				Name:      "keyring",
+				MountPath: "/var/lib/mysql-all-keyring",
```

Comment:
it's not good idea to mention "all"

Reply:
any suggestions? :)

Reply:
`/var/lib/mysql-keyring` is ok

Reply:
done https://github.com/percona/percona-xtradb-cluster-operator/pull/341/commits/a627074965333062b8f11ce029114ddad80b68f9